### PR TITLE
[OLS-393]  added labels in container file for konflux

### PR DESCRIPTION
**Pull request by xrajesh**
Added Labels in containerfile that is required for konflux release process

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
it looks ok

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,13 @@ RUN npm run build --loglevel verbose
 FROM registry.access.redhat.com/ubi9/nginx-120:latest
 
 COPY --from=build /usr/src/app/dist /usr/share/nginx/html
+LABEL name="openshift-lightspeed/lightspeed-console" \
+      com.redhat.component="openshift-lightspeed" \
+      io.k8s.display-name="OpenShift Lightspeed Console" \
+      summary="OpenShift Lightspeed Console provides OCP console plugin for Openshift Lightspeed Service" \
+      description="OpenShift Lightspeed Console provides OCP console plugin for Openshift Lightspeed Service " \
```
Comment (kyoto) [inline Dockerfile:16]:
```suggestion
      summary="OpenShift Lightspeed Console provides OCP console plugin for OpenShift Lightspeed Service" \
      description="OpenShift Lightspeed Console provides OCP console plugin for OpenShift Lightspeed Service" \
```

**Pull request merged**